Dockerfile

```diff
@@ -1,6 +1,10 @@
 ARG K_COMMIT
 FROM runtimeverificationinc/kframework-k:ubuntu-focal-${K_COMMIT}
 
+RUN    apt-get update                                   \
+    && apt-get install --yes software-properties-common \
+    && add-apt-repository ppa:ethereum/ethereum
+
 RUN    apt-get update            \
     && apt-get upgrade --yes     \
     && apt-get install --yes     \
```

Comment:
:sparkles: Consider merging the two `apt-get install` commands:

```dockerfile
RUN add-apt-repository ppa:ethereum/ethereum

RUN    apt-get update             \
    && apt-get upgrade --yes      \
    && apt-get install --yes      \
       ...
       software-properties-common \
       ...
```

Reply:
We can't actually, `add-apt-repository` depends on `software-properties-common` being installed, so that has to happen first so that we can install `solc` in the second block.